src/incomingDelivery/controller.js

```diff
@@ -23,6 +23,8 @@ router.post("/undoReceive", undoReceive);
 router.get("/allReceived", getAllReceived);
 router.get("/:deliveryId", getOne);
 
+router.put("/cancel", setCanceled);
```

Comment:
Might wanna add the checkId middleware here

Reply:
Done.